src/components/confusionMatrix/ConfusionMatrix.scss

```diff
@@ -4,7 +4,8 @@
 $row-color1: #e7ebf0;
```

Comment:
Give meaning to variables instead of using `var1`, `var2` etc

Reply:
It's not `var1` or `var2`, I guess that those names make sense on those variables

Reply:
But they don't reflect behavior

Reply:
Do you have any suggestion?